web-interface/vm-consoles-modules/vm-desktop-viewer-web.adoc

```diff
@@ -0,0 +1,15 @@
+[[vm-desktop-viewer-web]]
+== Using Desktop Viewer to connect to RDP Console (Windows only)
+
+Connect with *Desktop Viewer* to running virtual machine from the *Consoles* tab in the *Virtual Machine Details* screen of the web console.
```

Comment:
I'd suggest going with something like "The *Desktop Viewer* option can be used to connect to the RDP console of  a running Windows virtual machine." I don't think the specifics of where the option is located is necessary since the procedure is explained below it.

If you'd prefer to keep what you have, I'd change it to: "You can connect to a running virtual machine using the *Desktop Viewer* option from the *Consoles* tab in the *Virtual Machine Details* screen of the web console.